src/main/java/org/jenkinsci/plugins/casc/JenkinsConfigurator.java

```diff
@@ -51,13 +47,6 @@ public Jenkins configure(CNode c) throws ConfiguratorException {
 
         attributes.add(new PersistedListAttribute<Jenkins, Cloud>("clouds", Cloud.class)
             .getter(target -> target.clouds));
-        attributes.add(new MultivaluedAttribute<Jenkins, String>("jobs", String.class)
-            .setter((target, value) -> {
```

Comment:
how does it work after removing this bit?

Reply:
This is rootconfigurator, so just set 'jobs' as root element in yaml (we actually had a duplicate here)